entity-service/src/main/kotlin/com/egm/stellio/entity/model/Attribute.kt

```diff
@@ -47,28 +47,28 @@ open class Attribute(
     open fun serializeCoreProperties(): MutableMap<String, Any> {
         val resultEntity = mutableMapOf<String, Any>()
 
-        resultEntity[NgsiLdParsingUtils.NGSILD_CREATED_AT_PROPERTY] = mapOf(
-            NGSILD_ENTITY_TYPE to NGSILD_DATE_TIME_TYPE,
+        resultEntity[JsonLdUtils.NGSILD_CREATED_AT_PROPERTY] = mapOf(
```

Comment:
Strange to have static imports and this not statically imported

Reply:
Well, just some legacy (nothing forces you to static import and we have no defined rule about this)